### PR TITLE
Add warning system for battery level

### PR DESCRIPTION
**Pull request by callemand**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [x] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [x] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [x] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to the community ([french forum](https://community.gladysassistant.com/)/[english forum](https://en-community.gladysassistant.com/)) for testing before merging.

### Description of change

Screenshot 2023-11-14 at 11 10 47
Screenshot 2023-11-14 at 11 10 38

### CODE REVIEW

File: front/src/config/i18n/fr.json [hunk 1]
```diff
@@ -2053,7 +2053,9 @@
       "paused": "En Pause",
       "exited": "Arrêté",
       "dead": "Mort"
-    }
+    },
+    "batteryLevel": "Alerte sur le niveau de batterie",
+    "batteryLevelDescription": "A 09h00, uniquement le samedi, un message sera envoyé à tous les admins dès que le niveau du batterie parsera en dessous du pourcentage choisi"
```
Comment (Pierre-Gilles) [inline front/src/config/i18n/fr.json]: Typo: "parsera"

"admins" => "administrateurs" 

**Texte corrigé:** 

Tous les samedis à 9h00, un message sera envoyé à tous les administrateurs si le niveau de batterie d'un appareil passe en dessous du seuil choisi.

File: front/src/config/i18n/en.json [hunk 2]
```diff
@@ -2052,7 +2052,9 @@
       "paused": "Paused",
       "exited": "Exited",
       "dead": "Dead"
-    }
+    },
+    "batteryLevel": "Battery level alert",
+    "batteryLevelDescription": "At 09:00 AM, only on Saturday, a message will be sent to all admins as soon as the battery level drops below the chosen percentage"
```
Comment (Pierre-Gilles) [inline front/src/config/i18n/en.json]: Every Saturday at 9:00 am, a message will be sent to all administrators if a device's battery level falls below the chosen threshold.

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Typo & improvements on translations! Then it's good for me!

File: server/lib/device/device.checkBatteries.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  devices.forEach((device) => {
+    device.features
+      .filter((feature) => {
+        return feature.last_value < minPercentBattery;
+      })
+      .forEach((feature) => {
+        const messages = {
+          [AVAILABLE_LANGUAGES.EN]: `Warning !!! Battery level on ${device.name} is under ${minPercentBattery}% (current: ${feature.last_value}%)`,
+          [AVAILABLE_LANGUAGES.FR]: `Avertissement !!! Le niveau de la batterie de ${device.name} est inférieur à ${minPercentBattery} % (actuel : ${feature.last_value} %)`,
```
Comment (Pierre-Gilles) [inline server/lib/device/device.checkBatteries.js]: Can we put this in messages translation?

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]

File: server/lib/device/device.checkBatteries.js (hunk 3 above)
Comment (callemand) [reply server/lib/device/device.checkBatteries.js]: @Pierre-Gilles : Do you have any translation into server side ?

**Review by Pierre-Gilles — APPROVED** [review]
Good for me now! Thanks for this great PR 🙏

**Pull request merged**